test/ex_doc/autolink_test.exs

```diff
@@ -107,6 +107,9 @@ defmodule ExDoc.AutolinkTest do
       assert autolink("for/1", apps: [:elixir]) ==
                ~m"[`for/1`](Kernel.SpecialForms.html#for/1)"
 
+      assert autolink("for/1", app: :elixir) ==
+               ~m"[`for/1`](Kernel.SpecialForms.html#for/1)"
```

Comment:
Are you sure we need a new app: flag? If you see above, we were already using relative links if `apps: [:elixir]`. We just need to make sure the proper apps flag is set when generating Elixir docs.

Reply:
Thank you. There was a bug somewhere else. Can you please review the PR again?